### ISSUE TITLE
Docker-based install process requires people to clone repositories

### ISSUE THREAD

**Issue by ferrouswheel** [singnet/opencog-tutorials#9]
Most people expect to just pull a published image and have the requirements instead the image.

Do people really need to clone all the repos just to try a tutorial?

It'd be nicer to just run a `docker-compose.yml`, which refers to images on docker hub.

**Comment by pennachin** [comment]
Yeah, agreed. The plan is for that to happen once we have releases, so we can just use latest release images.
